.github/workflows/release.yaml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [main]
     tags:
       - "**" # Push events to every tag including hierarchical tags like v1.0/beta
+  pull_request:
+    branches: [main]
```

Comment:
```suggestion
```

Since `workflow_dispatch` can be executed by specifying a branch, there is no need to trigger `pull_request` here.
You can test manually with `workflow_dispatch`.